doc/customization_basic1.rst

```diff
@@ -0,0 +1,120 @@
+  After the initial run, it is recommended to set the retrieval of databundle and cost data to ``false`` to prevent unnecessary redownloading of data.
+
+
+Specify the weather year scope
+------------------------------
+With these arguments, it is possible to set the time horizon of the simulation
+Likewise, the example's temporal scope can be restricted (e.g. to 7 days):
+
+.. code:: yaml
+
+    snapshots:
+        start: "2013-03-01"
+        end: "2013-03-07"
+        inclusive: "left" # end is not inclusive
+
+.. note::
+
+    Ensure that the selected date range aligns with the dates available in the cutout dataset. If the weather data within the cutouts corresponds to the year 2013, then the range of snapshots should fall within that same year.
+
+Specify the demand year
+-----------------------
+
+This section specifies some parameters needed to generate demand profiles. 
+
+.. code:: yaml
+
+    load_options:
+      ssp: "ssp2-2.6"
+      weather_year: 2013
+      prediction_year: 2030
+      scale: 1
+
+The arguments you see are relative:
+- Weather_year: sets the year referenced by the weather data for calculating electricity demand profiles for the selected area.
+- Prediction_year: sets the Shared Socioeconomic Pathways (SSP) trajectory. Pypsa Earth uses the SSP2-2.6 scenario characterized by average challenges for mitigation and adaptation efforts to avoid global resgliding of about 2.6° by the end of the 21st century. Available values for weather_year and prediction_year can be checked by consulting the pypsa-earth/data/ssp2-2.6 folder
+	Currently, pre-calculated demand data are available for weather years 2011, 2013, 2018 and prediction years 2030, 2040, 2050 and 2100.
+
+Configure `atlite` section
+--------------------------
+
+To accurately model both temporally and spatially renewable availabilities such as wind and solar energy, historical climate data are processed with the atlite package.
+
+.. code:: yaml
+
+    atlite:
+        nprocesses: 4
+        cutouts:
+            cutout-2013-era5:
+                module: era5
+                dx: 0.3  # cutout resolution
+                dy: 0.3  # cutout resolution
+                # The cutout time is automatically set by the snapshot range.
+
+When you use precompiled cutouts, no editing of this section is required. 
+However, when using precompiled cutouts, you must replace all "cutout-2013-era5" entries with the name of the custom cutout.
+E.g.: if you simulate Kazakhstan with cutout: asia-2013-era5, each occurrence of cutout-2013-era5 should be updated to asia-2013-era5, which refers to the asia-2013-era5.nc file generated in the cutout folder.
+
+Configure `coordinates`
```

Comment:
Ok I see it is here, It may be good to move it at the beginning

Reply:
Perfect, no problem!